server/src/routes/api/event.mjs

```diff
@@ -39,10 +39,13 @@ function sendEvent(req, res) {
   }
 
   function fErr(method) {
+    const errorCode = (events.eventErrorType === 'validationError') ? 'EVENT-VALIDATION-FAILED' : 'NO-EVENT-HANDLER-REGISTERED';
     res.status(400).send({
       status: 'ERROR',
-      errorCode: 'NO-EVENT-HANDLER-REGISTERED',
-      message: `Could not send ${method} event because no listener is active`
+      errorCode: errorCode,
+      message: (errorCode === 'EVENT-VALIDATION-FAILED') ?
+        `Event validation failed for ${method}. Please ensure the event data meets the required format and try again.` :
```

Comment:
Minor/stylistic: I vote for lifting this ternary operator up out of the 'anonymous' object creation... having a new "const message = xxx ? yyy : zzz line and then just using message: message here (like you've done with errorCode)

Reply:
Addressed